<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_video" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="weightlabel">
       <property name="text">
        <string>宽</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_weight">
       <property name="text">
        <string>800</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hightlabel">
       <property name="text">
        <string>高</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_height">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_display">
       <property name="text">
        <string>播放</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>338</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="fpslabel">
       <property name="text">
        <string>帧率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fpsBox">
       <item>
        <property name="text">
         <string>25fps</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30fps</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="filelabel">
       <property name="text">
        <string>文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filepath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>511</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentFrame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>widget_video</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>currentFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
